pegasus/src/database.rb

```diff
@@ -5,10 +5,11 @@ class Tutorials
 
   def initialize(table)
     @table = table
+    @contents = DB[@table].all
```

Comment:
Good question.  Currently 61 rows, in the form of this csv: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/cache/Data_HocTutorials.gsheet.csv